nvflare/private/fed/server/server_engine.py

```diff
@@ -211,7 +212,14 @@ def wait_for_complete(self, job_id, process):
                     break
                 time.sleep(0.1)
             with self.lock:
-                return_code = process.poll()
+                run_dir = os.path.join(workspace, job_id)
+                rc_file = os.path.join(run_dir, FLMetaKey.PROCESS_RC_FILE)
+                if os.path.exists(rc_file):
+                    with open(rc_file, "r") as f:
+                        return_code = int(f.readline())
+                    os.remove(rc_file)
+                else:
+                    return_code = process.poll()
```

Comment:
This is getting the return code on server side.

Do we need to do the same thing on client side (client_executor.py)?

Reply:
good point. We do need to enhance the same on the client_executor. Added.